### PR TITLE
Bump cargo-component to 0.3.0.

### PR DESCRIPTION
**Pull request by peterhuene**
This also bumps wit to 0.2.0.

### CODE REVIEW

**Comment by peterhuene** [comment]
We'll merge this after #156.

**Comment by peterhuene** [comment]
Ignore the CI failure as I'll rebase this after #157 merges.

**Review by fibonacci1729 — APPROVED** [review]

**Review by elliottt — APPROVED** [review]
:tada:

**Pull request merged**
